scope_test.go

```diff
@@ -586,9 +586,10 @@ func TestEventProcessorsModifiesEvent(t *testing.T) {
 
 	if processedEvent == nil {
 		t.Error("event should not be dropped")
+	} else {
+		assertEqual(t, LevelFatal, processedEvent.Level)
+		assertEqual(t, []string{"wat"}, processedEvent.Fingerprint)
 	}
-	assertEqual(t, LevelFatal, processedEvent.Level)
-	assertEqual(t, []string{"wat"}, processedEvent.Fingerprint)
```

Comment:
The fix here is to use `t.Fatal` above. I had that in a local branch, merged to master now.

Reply:
LGTM, thanks!